packages/apply-release-plan/src/index.test.ts

```diff
@@ -2352,35 +2304,18 @@ describe("apply release plan", () => {
 
       let gitCmd = await spawn("git", ["status"], { cwd: tempDir });
 
-      expect(gitCmd.stdout.toString().includes("nothing to commit")).toBe(true);
+      const changesetsDeleted = releasePlan.changesets.reduce(
+        (prev, { id }) => {
+          return (
+            prev &&
+            gitCmd.stdout.toString().includes(`deleted:    .changeset/${id}.md`)
+          );
+        },
+        true as boolean
```

Comment:
This is already inferred correctly
```suggestion
        true
```